src/app/invest/invest.component.html

```diff
@@ -1,115 +1,128 @@
+                      Min. Investment
+                    </div>
+                    <div class="text-sm flex">
+                      <span class="font-semibold mr-1">{{ project.maxInvestment | currencyDefault }}</span>
+                      Max Investment
+                    </div>
+                  </div>
+                </div>
 
-        <div class="max-w-6xl w-full">
+                <!-- Wallet view section -->
+                <div class="lg:ml-8 mt-8 flex flex-col items-end bg-gray-200 p-4 rounded-2xl">
+                  <div class="text-lg">{{ project.walletBalance | currencyDefault }}</div>
+                  <div class="text-sm text-gray-500 whitespace-nowrap flex items-center mt-1">
+                    <svg width="20" height="20" viewBox="0 0 20 20" fill="none"
+                         xmlns="http://www.w3.org/2000/svg">
+                      <path
+                        d="M18.6087 1.78261C18.8246 1.78261 19 1.60721 19 1.3913C19 1.1754 18.8246 1 18.6087 1H2.95652C1.87776 1 1 1.87776 1 2.95652V17.0435C1 18.1222 1.87776 19 2.95652 19H18.6087C18.8246 19 19 18.8252 19 18.6087V4.52174C19 4.30583 18.8246 4.13044 18.6087 4.13044C17.9616 4.13044 17.4348 3.60366 17.4348 2.95652C17.4348 2.30938 17.9616 1.78261 18.6087 1.78261ZM2.95652 1.78261H17.0442C16.7985 2.10971 16.6522 2.5163 16.6522 2.95652C16.6522 3.39674 16.7985 3.80333 17.0442 4.13044H2.95652C2.30938 4.13044 1.78261 3.60366 1.78261 2.95652C1.78261 2.30938 2.30938 1.78261 2.95652 1.78261ZM18.2174 13.5217H11.5652C10.4865 13.5217 9.6087 12.644 9.6087 11.5652C9.6087 10.4865 10.4865 9.6087 11.5652 9.6087H18.2174V13.5217ZM18.2174 4.91304V8.82609H11.5652C10.0548 8.82609 8.82609 10.0548 8.82609 11.5652C8.82609 13.0752 10.0548 14.3043 11.5652 14.3043H18.2174V18.2174H2.95652C2.30938 18.2174 1.78261 17.6906 1.78261 17.0435V4.52097C2.10971 4.76669 2.5163 4.91304 2.95652 4.91304H18.2174Z"
+                        fill="black" stroke="#6B7280"/>
+                    </svg>
+                    <span class="ml-2">Available in wallet</span>
+                  </div>
+                </div>
+              </div>
 
-             <!-- Input investment card -->
-            <div [hidden]="(isInReview$ | async) === true" >
-                <div class="flex flex-col lg:max-h-80 lg:flex-row mt-8 shadow-lg rounded-3xl bg-white">
-                    <img class="lg:w-5/12 w-full rounded-t-3xl lg:rounded-l-3xl object-cover h-full card-corner-radius-override"
-                        src="{{ project.imgSrc }}"
-                        alt="">
-                    <div class="w-full flex flex-col p-4 lg:p-8">
-                        <form [formGroup]="investmentAmountForm" id="investmentAmountForm">
-    
-                            <!-- Input with label, min invest and max invest amounts -->
-                            <h1 class="pt-2 lg:pt-8 text-4xl uppercase">{{ project.title }}</h1>
-                            <div class="mt-4 flex flex-col lg:flex-row justify-between">
```

Comment:
in mobile screens, `block` class is a default value, so there is no need for `flex flex-col` (and then `lg:flex-row`). If you need a row on lg screens, you can use just `lg:flex`.